File: ISA_FRONTEND/src/app/model/address.ts
export class Address {
    id: string = '';
    country: string = '';
    city: string = '';
    street: string = '';
    streetNum: string = '';
    longitude: string = '';
    latitude: string = '';

    public constructor(obj?: any) {
        if (obj) {
            this.id = obj.id;
            this.country = obj.country;
            this.city = obj.city;
            this.street = obj.street;
            this.streetNum = obj.streetNum;
            this.longitude = obj.longitude;
            this.latitude = obj.latitude;
        }
    }
}

File: ISA_BACKEND/BloodBank/src/main/java/com/example/bloodbank/Model/Address.java
package com.example.bloodbank.Model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "Addresses")
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Address {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private long Id;
    @Column
    private String Country;
    @Column
    private String City;
    @Column
    private String Street;
    @Column
    private String StreetNum;
    @Column
    private String Longitude;
    @Column
    private String Latitude;
}
